src/js/components/Audio/Audio.js

```diff
@@ -0,0 +1,276 @@
+    clearTimeout(this.interactionTimer);
+    this.interactionTimer = setTimeout(this.interactionStop, 3000);
+  };
+
+  interactionStop = () => {
+    const { focus } = this.state;
+    if (!focus && !this.unmounted) {
+      this.setState({ interacting: false });
+    }
+  };
+
+  setVolume = value => {
+    const { audioRef } = this.state;
+    audioRef.current.volume = value;
+    this.setState({ volumeValue: value });
+  };
+
+  seek = value => {
+    const { audioRef, duration } = this.state;
+
+    if (audioRef.current) {
+      const precentageProgress = value * duration;
+      audioRef.current.currentTime = precentageProgress;
+      this.setState({ scrubberValue: value });
+    }
+  };
+
+  renderControls() {
+    const { theme } = this.props;
+    const {
+      currentTime,
+      duration,
+      interacting,
+      playing,
+      scrubberValue,
+      volumeValue,
+    } = this.state;
+
+    const background = (theme.audio.controls &&
+      theme.audio.controls.background) || {
+      color: 'dark-1',
+      opacity: 'strong',
+    };
+
+    const durationFormattedTime = formatTime(duration);
+    const scrubFormattedTime = formatTime(currentTime);
+
+    const Icons = {
+      Pause: theme.audio.icons.pause,
+      Play: theme.audio.icons.play,
+      Volume: theme.audio.icons.volume,
+    };
+
+    return (
+      <StyledAudioControls active={interacting}>
+        <Box direction="column" background={background}>
+          <RangeInput
+            min={0}
+            max={1}
+            step={this.scrubberStep}
+            value={scrubberValue}
+            onChange={event => this.seek(event.target.value)}
+          />
+          <Box direction="row" align="center" justify="between">
+            <Box align="center" direction="row">
+              <Button
+                icon={
+                  playing ? (
+                    <Icons.Pause color={theme.audio.icons.color} />
+                  ) : (
+                    <Icons.Play color={theme.audio.icons.color} />
+                  )
+                }
+                hoverIndicator="background"
+                onClick={playing ? this.pause : this.play}
+              />
+              <Box pad={{ horizontal: 'xsmall' }} direction="row" gap="xsmall">
+                <Text textAlign="center" margin="none">
+                  {scrubFormattedTime || '00:00'}
+                </Text>
+                <Text
+                  textAlign="center"
+                  margin="none"
+                  color={theme.audio.controls.duration.color}
+                >
+                  {durationFormattedTime}
+                </Text>
+              </Box>
+            </Box>
+            <Box
+              pad={{ horizontal: 'xsmall' }}
+              direction="row"
+              align="center"
+              gap="xsmall"
+            >
+              {/* TODO make hover to show hide on responsive  */}
+              <Icons.Volume color={theme.audio.icons.color} />
```

Comment:
I'd explore making the volume control just a button to start with and then when the user clicks it shows the range input for volume, taking over the whole footer.

Reply:
do you think it will be better if it happens on `hover` (e.g. youtube) or only after clicking it?

Reply:
We tend to avoid hover as a primary means of interaction as it doesn't work on touch devices.